src/components/MenuPanel/MenuPanel.js

```diff
@@ -53,7 +59,7 @@ export class MenuPanel extends PureComponent<Props> {
           isMnemonic={isMnemonic}
         />
         <Separator />
-        <Actions />
+        <Actions routeName={routeName} />
```

Comment:
The same hack is required for Settings. :( Are you sure you want to put them into separate components?

Reply:
no need to do it for settings